kilian.py

```diff
@@ -156,6 +159,114 @@ async def studid(ctx: interactions.CommandContext, member: interactions.Member):
         await ctx.send(database.get_matr_nr(str(member.id)), ephemeral=True)
 
 
+    @bot.command()
+    @interactions.option(description="Course chat you want to join.")
+    async def join(ctx: interactions.CommandContext, course: interactions.Role):
+        """Join a course chat."""
+        role_id = str(course.id)
+        guild_id = str(ctx.guild_id)
+
+        if not database.is_kusss(str(ctx.author.id)):
+            await ctx.send("You have to be subscribed to Kilian services to use Kilian features.", ephemeral=True)
+            return
+
+        if not database.is_managed_role(guild_id, role_id):
+            await ctx.send("This course does not exist!", ephemeral=True)
+            return
+
+        discord_id = str(ctx.author.id)
+        semester = uni.current_semester()
+        lva_name = database.get_lva_name_by_role_id(semester, guild_id, role_id)
+        lva_nr = database.get_lva_nr(lva_name, semester)
+
+        if database.student_has_course(discord_id, semester, lva_name):
+            await ctx.send("Already joined the course chat.", ephemeral=True)
+            return
+
+        student = uni.student(discord_id, database.get_link(discord_id))
+        student_lva_nrs = set(map(lambda c: c.lva_nr, student.courses))
+        course_lva_nrs = database.get_lva_nrs(lva_name, semester)
+
+        intersection = student_lva_nrs & course_lva_nrs
+
+        if len(intersection) > 0:
+            database.insert(StudentCourse(discord_id, semester, lva_nr, 1))
```

Comment:
@xMuffin, add all lva_nr from intersection here instead of the smallest(random) one, so the student_courses actually represent the real courses on a rejoin

something like 
```python
for lva_nr_runner in intersection:
    database.insert(StudentCourse(discord_id, semester, lva_nr_runner, 1))
```
should work

Reply:
If "active" from "Studentcourse" is a boolean value, please use True and False. I know Python allows this, but hell nah!